101-dataset-create-tabular-from-relative-path/azuredeploy.json

```diff
@@ -0,0 +1,151 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"workspaceName": {
+			"type": "string",
+			"metadata": {
+				"description": "Specifies the name of the Azure Machine Learning workspace which will hold this datastore target."
+			}
+		},
+		"datasetName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the dataset."
+			}
+		},
+		"datasetDescription": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description": "Optional : The description for the dataset."
+			}
+		},
+		"datastoreName": {
+			"type": "string",
+			"metadata": {
+				"description": "The datastore name."
+			}
+		},
+		"relativePath": {
+			"type": "string",
+			"metadata": {
+				"description": "Path within the datastore"
+			}
+		},
+		"sourceType": {
+			"type": "string",
+			"defaultValue": "delimited_files",
+			"allowedValues": [
+				"delimited_files",
+				"json_lines_files",
+				"parquet_files"
+			],
+			"metadata": {
+				"description": "Data source type"
+			}
+		},
+		"separator": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description": "Optional: The separator used to split columns for 'delimited_files' sourceType, default to ',' for 'delimited_files'"
+			}
+		},
+		"header": {
+			"type": "string",
+			"defaultValue": "all_files_have_same_headers",
+			"allowedValues": [
+				"all_files_have_same_headers",
+				"only_first_file_has_headers",
+				"no_headers",
+				"combine_all_files_headers"
+			],
+			"metadata": {
+				"description": "Optional :  Header type. Defaults to 'all_files_have_same_headers'"
+			}
+		},
+		"partitionFormat": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description": " Optional : The partition information of each path will be extracted into columns based on the specified format. Format part '{column_name}' creates string column, and '{column_name:yyyy/MM/dd/HH/mm/ss}' creates datetime column, where 'yyyy', 'MM', 'dd', 'HH', 'mm' and 'ss' are used to extract year, month, day, hour, minute and second for the datetime type. The format should start from the position of first partition key until the end of file path. For example, given the path '../USA/2019/01/01/data.parquet' where the partition is by country/region and time, partition_format='/{CountryOrRegion}/{PartitionDate:yyyy/MM/dd}/data.csv' creates a string column'CountryOrRegion' with the value 'USA' and a datetime column 'PartitionDate' with the value '2019-01-01"
```

Comment:
seems like we should move this to the readme or documentation (with a pointer) - aside from source code, I don't think there will be many experiences where this will be consumable.

Reply:
Actually we don't do a good job of documenting this publicly so pointer is not available. We show similar tool tip in UI to educate our customers about this. Could we please keep it here to be more accessible while deploying template in Azure? 

![parrtitionformat](https://user-images.githubusercontent.com/12025414/87689920-a564ca80-c73d-11ea-9a95-044a4418020d.jpg)